r/tools/nixlibs.R

```diff
@@ -423,13 +423,14 @@ cmake_version <- function(cmd = "cmake") {
   )
 }
 
-turn_off_thirdparty_features <- function(env_var_list) {
+turn_off_optional_features <- function(env_var_list) {
```

Comment:
This is probably just 🚲 🏠 ing, but I wonder if we should name this `turn_off_all_optional_features()` so it's clear that we intend everything optional be be disabled, and we're not picking and choosing some (like we do elsewhere in this script)